packages/terra-date-time-picker/tests/nightwatch/date-time-picker-spec.js

```diff
@@ -7,7 +7,7 @@ const timeZones = ['America/Chicago', 'America/Denver', 'America/Detroit', 'Amer
   'America/Los_Angeles', 'America/New_York', 'America/Phoenix'];
 const includesTZ = timeZones.includes(moment.tz.guess());
 
-module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous'], {
+module.exports = resizeTo(['tiny', 'medium', 'huge'], {
```

Comment:
Is there a need for 3 breakpoitns here? Wondering if we can just do tiny and large.

Reply:
Updated: https://github.com/cerner/terra-core/pull/1166/commits/f527b4d8133d9be4b6f75715c93a7bf3ec3550be